ci/docker/ml.build.Dockerfile

```diff
@@ -1,5 +1,7 @@
-ARG DOCKER_IMAGE_BASE_BUILD=cr.ray.io/rayproject/oss-ci-base_ml
-FROM $DOCKER_IMAGE_BASE_BUILD
+ARG BASE_BUILD
+FROM cr.ray.io/rayproject/$BASE_BUILD
```

Comment:
why fortify the prefix? I like the original `DOCKER_IMAGE_BASE_BUILD`, which is more flexible?

Reply:
It's because of line 21, the name of the base image cannot be really free form anymore since its name now has some meaning. Or I can add a another arg IS_GPU to decouple the name and its meaning.